src/common_h.py

```diff
@@ -0,0 +1,196 @@
+# -*- coding: utf-8 -*-
+#
+# libocispec - a C library for parsing OCI spec files.
+#
+# Copyright (C) Huawei Technologies., Ltd. 2018-2020. All rights reserved.
```

Comment:
could you please also add the original copyright line to these new files since they are based on code from generate.py?

Reply:
fixed